README.md

```diff
@@ -557,6 +557,41 @@ application: litestar.Litestar = (
 >
 > In this case, Prometheus will receive `{"first_value": 1, "second_value": 2}` inside `prometheus_additional_params`. This is also true for `list`, `tuple`, and `set`.
 
+### Using microbootstrap without a framework
+
+You can use microbootstrap to set up Sentry, OpenTelemetry and Logging with `InstrumentsSetupper`:
```

Comment:
I can, but for what purpose? I cant understand from this example

Reply:
Just add a job/consumer small example